packages/react/src/VideoPlayer/VideoPlayer.stories.tsx

```diff
@@ -56,12 +56,14 @@ export default {
       type: 'boolean',
     },
   },
-} as Meta<typeof VideoPlayer>
+}
+
+export default meta
 
 export const Playground: StoryFn<typeof VideoPlayer> = args => (
   <VideoPlayer {...args}>
-    <VideoPlayer.Source src="https://primer.github.io/brand/assets/example.mp4" type="video/mp4" />
-    <VideoPlayer.Track src="https://primer.github.io/brand/assets/example.vtt" default />
+    <VideoPlayer.Source src="/brand/storybook/example.mp4" type="video/mp4" />
```

Comment:
This example isn't working in local environments and assets are 404'ing as the URL assumes you're viewing in staging or prod...

![Screenshot 2024-07-31 at 17 27 05](https://github.com/user-attachments/assets/f4399bbb-6416-436c-8012-b2b3aa183951)

Can we roll back any changes that aren't specific to the upgrade to rule out regressions like this? We can patch them up in a follow-up PR.

Reply:
That's rolled back, I'll take a look in a subsequent PR